src/classes/RD_RecurringDonations.cls

```diff
@@ -1128,21 +1139,29 @@ public class RD_RecurringDonations {
     * @param r recurring donation record
     * @return date First date to use for created Donations
     */
-    private static date getStartDate(npe03__Recurring_Donation__c r) {
-        Date startDt;
-        if (r.npe03__Next_Payment_Date__c != null)
-            startDt = r.npe03__Next_Payment_Date__c;
-        else {
-            startDt = r.npe03__Date_Established__c;
-        }
+    @TestVisible
+    private static Date getStartDate(npe03__Recurring_Donation__c r) {
```

Comment:
@force2b , functional changes are made in the `getStartDate()` method. Other changes are made to comply the code with the latest coding standard. Let me know if I should remove the style changes completely from this PR.

Reply:
Thanks @RadSFDeveloper. This is good. Just needed help finding the right method.